topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1085 @@
+
+
+
+*  a 49-sample simulated metagenome data from Human Microbiome data to represent five different body sites of the human host, namely gastrointestinal tract, oral cavity, airways, skin and urogenital tract.
+
+> ### {% icon details %} Human microbiom dataset details
+>
+> <iframe src="https://docs.google.com/spreadsheets/d/e/2PACX-1vQgJr3J-IyVy9IkXS9W-RZcV83Tr6f7RusG_97QwgpW2dFdCXUMroROIhy8gKjPcUgISFXW9NQwOzzK/pubhtml?gid=503333420&amp;single=true&amp;widget=true&amp;headers=false" width="900" height="600" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
+>
+{: .details}
+
+*  a 64-sample simulated metagenome data from the guts of different mice, vendors and positions in the gut. (**[https://openstack.cebitec.uni-bielefeld.de:8080/swift/v1/CAMISIM_MOUSEGUT](https://openstack.cebitec.uni-bielefeld.de:8080/swift/v1/CAMISIM_MOUSEGUT)**)
+    * According to the tutorial [[20]](https://www.nature.com/articles/s41596-020-00480-3) the mouse gut metagenome toy dataset was generated with CAMISIM v.0.2 [[21]](https://pubmed.ncbi.nlm.nih.gov/30736849/) using a microbial community genome abundance distribution modelled from 791 public prokaryotic genomes marked as at least ‘scaffolds’ in the NCBI RefSeq [[22]](https://pubmed.ncbi.nlm.nih.gov/17130148/). They comprise 8 phyla, 18 classes, 26 orders, 50 families, 157 genera, and 549 species. The community genome abundance distribution matches as closely as possible the 16S taxonomic profiles for 64 mouse gut samples. As such, this dataset enables us to assess how well sequenced community members can be characterised with different techniques from the metagenomes of similar communities. On average, within each of the 64 samples, 91.8 genomes are represented. Both long- and short-read metagenomic sequencing data are available. The runtime to generate this data was approximately 3 weeks using eight CPU cores of a computer with an AMD Opteron 6378 CPU and 968 GB of main memory.
+
+Both include  5 Gb long paired-end reads (Pacific Biosciences, variable length with a mean of 3000 bp) and 5 Gb short paired-end reads (Illumina HiSeq2000, 150 bp).
+
+
+##### Challenge datasets
+
+For the assembly challenge itself, 4 “challenge” datasets were offered: 
+
+
+
+* clinical pathogen detection,
+* marine, 
+* strain madness, 
+* plant associated data. 
+
+
+###### Clinical Pathogen detection
+
+The clinical pathogen detection dataset, a short-read metagenomic sequencing dataset of a blood sample from a patient with hemorrhagic fever of unknown cause, was excluded because it is provided only for the last challenge: identify a causal pathogen together with further pathogens.
+
+> ### {% icon details %} Clinical Pathogen detection dataset links
```

Comment:
same here, I think, just link text with the url you're interested in, rather than using details boxes with iframes. (This same comment applies to the rest of the datasets you discuss / details boxes with iframes.)